flowey/flowey_lib_hvlite/src/download_openhcl_kernel_package.rs

```diff
@@ -78,36 +78,36 @@ impl FlowNode for Node {
 
         for ((kind, arch), out_vars) in reqs {
             let version = versions.get(&kind).expect("checked above");
-            let kind_string = match kind {
-                OpenhclKernelPackageKind::Main | OpenhclKernelPackageKind::Cvm => "main",
-                OpenhclKernelPackageKind::Dev | OpenhclKernelPackageKind::CvmDev => "dev",
-            };
-
-            let tag = format!("rolling-lts/hcl-{kind_string}/{version}");
+            let tag = format!(
+                "rolling-lts/hcl-{}/{}",
+                match kind {
+                    OpenhclKernelPackageKind::Main | OpenhclKernelPackageKind::Cvm => "main",
+                    OpenhclKernelPackageKind::Dev | OpenhclKernelPackageKind::CvmDev => "dev",
+                },
+                version
+            );
 
             let file_name = format!(
-                "Microsoft.OHCL.Kernel.{}-{}-{}.{}",
-                version,
+                "Microsoft.OHCL.Kernel{}.{}{}-{}.tar.gz",
                 match kind {
-                    OpenhclKernelPackageKind::Main | OpenhclKernelPackageKind::Dev => {
-                        kind_string.to_string()
+                    OpenhclKernelPackageKind::Main | OpenhclKernelPackageKind::Cvm => {
+                        ""
                     }
-                    OpenhclKernelPackageKind::Cvm | OpenhclKernelPackageKind::CvmDev => {
-                        format!("{kind_string}-cvm")
+                    OpenhclKernelPackageKind::Dev | OpenhclKernelPackageKind::CvmDev => {
+                        ".Dev"
                     }
                 },
+                version,
+                match kind {
```

Comment:
When the next main kernel is updated this part becomes more simple (no more adding -main).